client/src/components/landing/Phases/ProcessCarousel.css

```diff
@@ -1,38 +1,42 @@
+.swiper-slide p {
```

Comment:
<img width="1433" alt="Screen Shot 2024-12-13 at 18 42 54" src="https://github.com/user-attachments/assets/60f943e6-cb5a-48e4-bfaf-0fd587fe9338" />

the text is out of the carousel, moreover the size of image in this section is not good. have you changed that?

Reply:
didnt change the dimension of images, as said in the first comment, we are going to replace descriptions and images. we have to do a meeting about that